pay-api/src/pay_api/services/receipt.py

```diff
@@ -200,3 +201,25 @@ def get_receipt_details(filing_data, invoice_identifier, skip_auth_check):
         # Format date to display in report.
         receipt_details['invoice']['createdOn'] = get_local_formatted_date(invoice_data.created_on)
         return receipt_details
+
+    @staticmethod
+    def get_nsf_receipt_details(payment_id):
+        """Return NSF receipt details, which can contain multiple invoices, combine these in PLI."""
+        receipt_details: dict = {}
+        invoices = Invoice.find_invoices_for_payment(payment_id, InvoiceReferenceStatus.COMPLETED.value)
```

Comment:
InvoiceStatus and not InvoiceReferenceStatus?

Reply:
needs to be invoice reference status, it was looking for ACTIVE by default before